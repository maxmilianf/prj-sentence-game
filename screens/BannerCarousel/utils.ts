export const gradientBackground = ['#a18cd1', '#fbc2eb'];
export const prevButtonName = 'chevron-left-box';
export const nextButtonName = 'chevron-right-box';
