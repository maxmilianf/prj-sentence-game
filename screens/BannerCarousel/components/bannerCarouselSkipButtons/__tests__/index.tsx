import React from 'react';
import renderer from 'react-test-renderer';
import { prevButtonName } from '../../../utils';

import BannerCarouselSkipButton from '../index';

it('BannerCarousel Skip buttons render correctly', () => {
  const tree = renderer
    .create(
      <BannerCarouselSkipButton
        iconName={prevButtonName}
      ></BannerCarouselSkipButton>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
