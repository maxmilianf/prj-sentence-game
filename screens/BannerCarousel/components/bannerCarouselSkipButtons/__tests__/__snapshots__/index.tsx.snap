// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerCarousel Skip buttons render correctly 1`] = `
<View
  accessibilityRole="button"
  accessibilityState={
    Object {
      "disabled": true,
    }
  }
  accessible={true}
  focusable={false}
  hitSlop={
    Object {
      "bottom": 6,
      "left": 6,
      "right": 6,
      "top": 6,
    }
  }
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "overflow": "hidden",
      },
      Array [
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "margin": 6,
          "overflow": "hidden",
        },
        Object {
          "borderRadius": 22.5,
          "height": 45,
          "width": 45,
        },
        undefined,
        Object {
          "opacity": 0.8,
        },
      ],
    ]
  }
>
  <View>
    <Text />
  </View>
</View>
`;
